.eslintrc.yaml

```diff
@@ -102,17 +102,17 @@ rules:
   func-name-matching: error
   func-style: [error, declaration, {allowArrowFunctions: true}]
   # indent: [error, 2, {ArrayExpression: first,
-  #                 CallExpression: {arguments: first},
-  #                 FunctionDeclaration: {parameters: first},
-  #                 FunctionExpression: {parameters: first},
-  #                 MemberExpression: off,
-  #                 ObjectExpression: first,
-  #                 SwitchCase: 1}]
+  #                     CallExpression: {arguments: first},
```

Comment:
This will conflict with https://github.com/nodejs/node/pull/14403.
IMHO both seem appropriate for this change...